src/nimiSlides.nim

```diff
@@ -541,15 +541,24 @@ template align*(text: string, body: untyped) =
   body
   nbRawHtml: "</div>"
 
+#templates can't have default args and untyped args at the same time
+#so we use overloading to get the same effect
+
+template columns*(columnGap: float, body: untyped) =
+  #tempted to use fmt"", but strformat doesn't support template args in the format string
+  nbRawHtml: """<div style="display: grid; grid-auto-flow: column; grid-auto-columns: minmax(0, 1fr); overflow-wrap: break-word; column-gap: """ & $columnGap & """em;">"""
+  body
+
 template columns*(body: untyped) =
-  nbRawHtml: """<div style="display: grid; grid-auto-flow: column; grid-auto-columns: minmax(0, 1fr); overflow-wrap: break-word;">"""
+  columns(0.0,body)
```

Comment:
From what I can [read](https://www.w3schools.com/CSSref/css3_pr_column-gap.php), the default value for `column-gap` should be 1em and not 0em.